src/cmdline.rs

```diff
@@ -418,6 +418,9 @@ pub struct CommonCustomizeConfig {
     /// NetworkManager keyfile for live & dest
     #[structopt(long, number_of_values = 1, value_name = "path")]
     pub network_keyfile: Vec<String>,
+    /// PEM cert authority bundle for Ignition
```

Comment:
For consistency and clarity, should this end with "for live & dest" ? E.g. "Ignition PEM cert authority bundle for live & dest".

Reply:
That won't fit, but we can do `Ignition PEM CA bundle for live & dest`.